src/DssSpell.sol

```diff
@@ -41,35 +55,172 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
+    uint256 constant THREE_PCT_RATE  = 1000000000937303470807876289;
 
+    uint256 constant MILLION    = 10**6;
     uint256 constant WAD        = 10**18;
     uint256 constant RAD        = 10**45;
-    uint256 constant MILLION    = 10**6;
 
-    address constant LERP = 0x7b3799b30f268BA55f926d7F714a3001aF89d359;
-
-    // Turn off office hours
-    function officeHours() public override returns (bool) {
-        return false;
-    }
+    address constant RWA001_OPERATOR           = 0x7709f0840097170E5cB1F8c890AcB8601d73b35f;
+    address constant RWA001_GEM                = 0x10b2aA5D77Aa6484886d8e244f0686aB319a270d;
+    address constant MCD_JOIN_RWA001_A         = 0x476b81c12Dc71EDfad1F64B9E07CaA60F4b156E2;
+    address constant RWA001_A_URN              = 0xa3342059BcDcFA57a13b12a35eD4BBE59B873005;
+    address constant RWA001_A_INPUT_CONDUIT    = 0x486C85e2bb9801d14f6A8fdb78F5108a0fd932f2;
+    address constant RWA001_A_OUTPUT_CONDUIT   = 0xb3eFb912e1cbC0B26FC17388Dd433Cecd2206C3d;
+    address constant MIP21_LIQUIDATION_ORACLE  = 0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30;
+    address constant SC_DOMAIN_DEPLOYER_07     = 0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf;
 
     function actions() public override {
-        VatAbstract vat = VatAbstract(DssExecLib.vat());
 
-        // De-authorize the lerp contract from adjusting the PSM-USDC-A DC
-        DssExecLib.deauthorize(address(vat), LERP);
+        // Increase ETH-A target available debt (gap) from 30M to 80M
+        DssExecLib.setIlkAutoLineParameters("ETH-A", 2_500 * MILLION, 80 * MILLION, 12 hours);
```

Comment:
This is a huge amount of DC risk every 12 hours.  This really should be smaller increments more frequently.   Not a blocker, but this isn't a great stage to find out about governance changes that violate the intent of the module.

Reply:
Yeah totally agree. I feel nobody in the community and risk actually understood the main purpose of this module.

Reply:
I can try to arrange a meeting for us with Risk